package.json

```diff
@@ -46,9 +46,13 @@
     "test-rooms-disabled": "ENABLE_ROOMS=false HIDDEN_ROUTES=/rooms mocha -R dot --recursive tests/roomFlag/roomFlagDisabled",
     "test-db": "npm run checkdb && mocha -R dot --recursive tests/db",
     "test-cors": "CORS_ROUTES=/v1/bots,/v1/botActions,/v1/botData,/v1/events,/v1/rooms mocha -R dot --recursive tests/cors",
-    "test-cache": "mocha -R dot --recursive tests/cache",
+    "test-jobQueue": "mocha --recursive tests/jobQueue",
+    "test-cache": "npm run test-cache-jobQueue && npm run test-cache-models && npm run test-cache-misc",
+    "test-cache-jobQueue": "mocha --recursive tests/cache/jobQueue",
+    "test-cache-models": "mocha --recursive tests/cache/models",
+    "test-cache-misc": "mocha tests/cache",
     "test-view": "NODE_ENV=build mocha -R dot --recursive --compilers js:babel-core/register --require ./tests/view/setup.js tests/view",
-    "test": "istanbul cover ./node_modules/mocha/bin/_mocha --report lcovonly -- -R dot --recursive tests/api tests/cache tests/clock tests/config tests/db tests/enforceToken tests/jobQueue tests/realtime tests/enableCache tests/logging && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage && npm run test-view && npm run test-disablehttp && npm run test-rooms-enabled && npm run test-rooms-disabled && npm run test-cors && npm run test-limiter",
```

Comment:
This is necessary because these scripts run tests from the same file, and you can't run tests twice in the same mocha process. See comments in test files.